campaign.html

```diff
@@ -77,9 +78,13 @@
   To find out more about options for becoming a partner company, like <a href="http://travis-ci.com">Travis CI</a> and <a href="http://github.com">GitHub</a>, please <a href="mailto:summer-of-code@railsgirls.com.">contact us</a>.
 </p>
 
+<div class="resize">
+  <img src="img/blog/2016/2016-04-21-thank-you-and-welcome-more-women-to-tech-01.jpg">
```

Comment:
the image source path should have a `../` in front of it; when the path is `../img/blog/ETC..` it works :)